config/settings/base.py

```diff
@@ -259,8 +261,26 @@
     "version": 1,
     "disable_existing_loggers": False,
     "formatters": {"verbose": {"format": "%(levelname)s %(asctime)s %(module)s " "%(process)d %(thread)d %(message)s"}},
-    "handlers": {"console": {"level": "DEBUG", "class": "logging.StreamHandler", "formatter": "verbose",}},
-    "root": {"level": "INFO", "handlers": ["console"]},
+    "handlers": {"console": {"level": "DEBUG", "class": "logging.StreamHandler", "formatter": "verbose",},
+    'db': { 'level': 'INFO', 'class': 'automated_logging.handlers.DatabaseHandler', }},
+    'loggers': { 'automated_logging': { 'level': 'INFO', 'handlers': ['db'], 'propagate': True, },
+    'django': { 'level': 'INFO', 'handlers': ['db'], 'propagate': True, },
+    }    
+}
+
+from logging import INFO
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "from logging import INFO" should be placed before "import environ"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5320498)